consul/handler.py

```diff
@@ -23,6 +23,7 @@
 DTFORMAT = "%Y-%m-%dT%H%M%S.%f"
 DEPLOY = '/deploy'
 CADDYLOGS = '/var/log'
+BTRFSDRIVER = os.environ['BTRFSDRIVER']
```

Comment:
could be nice to provide a default value likes:

``
BTRFSDRIVER = os.environ.get('BTRFSDRIVER', 'anybox/buttervolume:latest')
``

Reply:
You take care of this comment  above but you should remove this line which make [travis unhappy](https://travis-ci.org/mlfmonde/cluster/builds/377346022#L1139)

Actually I apologies I think I've made this duplication trough [that PR](https://github.com/ccomb/cluster/pull/1)